09-schema/01-concepts.md

```diff
@@ -518,7 +518,7 @@ The data types available in a Grakn knowledge graph are:
 - `double`: a double-precision floating point number, including a decimal point.
 - `string`: enclosed in double `"` or single `'` quotes
 - `boolean`: `true` or `false`
-- `date`: a date or date-time in ISO 8601 format
+- `date`: a date or date-time in format of `2005-11-06` or `2005-11-06T19:27:03.186`. The second and second fraction part can be optional. The second fraction part can have 1 to 3 digits.
```

Comment:
the explanation is a bit confusing. does our datetime format differ from ISO 8601 in that ours cannot contain timezone information?

Reply:
@sorsaffari No it's more than that. [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) seems to be a complex format, and we pretty much don't support any of the basics.

Reply:
I have read through ISO 8601 documentation and it seems to me the only part of the standard that we don't (yet) support is the timezone information

here is my suggestion for a clear and unambiguous explanation:

```
`date`: a date or date-time in ISO 8601 format excluding timezone information. 

<div class="note">
Acceptable date-time formats are:
- `yyyy-mm-ddThh:mm`
- `yyyy-mm-ddThh:mm:ss`
- `yyyy-mm-ddThh:mm:ss.s`
- `yyyy-mm-ddThh:mm:ss.ss`
- `yyyy-mm-ddThh:mm:ss.sss`

Acceptable date format is: `yyyy-mm-dd`
</div>
```

Reply:
Seems there's a basic format in ISO 8601 so you can have this `20200504T153234Z`

Reply:
@lriuui0x0 okay, let's not mention ISO 8601 as you are pointing that it can be misleading. 

do you have a better alternative for "The second and second fraction part can be optional. The second fraction part can have 1 to 3 digits."? sorry, man. I think that's just bad writing, especially for public documentation.

how about the snippet I shared in my last comment - the acceptable formats for `date` and `date-time`?

Reply:
@sorsaffari Yes I think that's good. I will apply them.